include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -16,21 +16,8 @@
 #ifndef _ONEDPL_CONFIG_H
 #define _ONEDPL_CONFIG_H
 
-#ifndef _PSTL_VERSION
-#    define _PSTL_VERSION 11000
-#    define _PSTL_VERSION_MAJOR (_PSTL_VERSION / 1000)
-#    define _PSTL_VERSION_MINOR ((_PSTL_VERSION % 1000) / 10)
-#    define _PSTL_VERSION_PATCH (_PSTL_VERSION % 10)
-#endif
-
-#define ONEDPL_VERSION_MAJOR 2022
-#define ONEDPL_VERSION_MINOR 6
-#define ONEDPL_VERSION_PATCH 0
-
-#if _ONEDPL___cplusplus >= 202002L && __has_include(<version>)
-#    include <version>
-#    define _ONEDPL_STD_FEATURE_MACROS_PRESENT 1
-#endif
+#include "../version"
```

Comment:
To me this line looks like the indication that we want to move `onedpl_config.h` one level higher in the directory hierarchy. Of course, I don't propose to do it in this PR but having it within `pstl` folder confuses me.

Reply:
One level upper are only public headers, and the config header does not belong there.

Actually, including the new version header into the config is just the easiest way to ensure two things:
- The version information and feature macros are available to all headers that may need those;
- The content of the standard `<version>` header is available for the config.
It can be done in a different way, for example, by splitting our version header into the public and internal ones.